core/src/main/java/org/apache/iceberg/PartitionsTable.java

```diff
@@ -275,6 +283,7 @@ static class Partition {
     private int eqDeleteFileCount;
     private Long lastUpdatedMs;
     private Long lastUpdatedSnapshotId;
+    private long dataFileSizeInBytes;
```

Comment:
nit: can we move after dataFileCount?  (as its part of 'dataFile' group)

Reply:
speak of which, @szehon-ho do you feel we shall do the same in Schema method to move this new field with id 11 to be right after file_count (field id 3)? It seem to fit into same dataFile group by it might be some concern about reference by position to mess up?

Reply:
Oh yea , i think that was the consensus from the other comemnt: https://github.com/apache/iceberg/pull/7920#discussion_r1244089947  @hsiang-c do you think we can move it?

Reply:
@szehon-ho Sure thing.